modules/frontend/cache_keys.go

```diff
@@ -15,25 +16,29 @@ const (
 )
 
 func searchJobCacheKey(tenant string, queryHash uint64, start int64, end int64, meta *backend.BlockMeta, startPage, pagesToSearch int) string {
-	return cacheKey(cacheKeyPrefixSearchJob, tenant, queryHash, start, end, meta, startPage, pagesToSearch)
+	startTime := time.Unix(start, 0)
```

Comment:
how do we know we are using seconds here? Mabe we can rename the arguments to

startSecods and endSeconds or smth similar

Reply:
pushed time.Time out one more level. i did the translation here b/c it seems like it has enough context to know if it's unix epoch seconds or nanos, but agree it's nicer to go one more level.